pkg/virt-handler/vm.go

```diff
@@ -1938,6 +1943,16 @@ func getVMIPodNetwork(vmi *v1.VirtualMachineInstance) *v1.Network {
 	return podNetwork
 }
 
+func (d *VirtualMachineController) checkSRIOVInterfacesForMigration(vmi *v1.VirtualMachineInstance) error {
```

Comment:
Can you use `validate` instead of `check` here as well?

Reply:
Done